data/sql/updates/pending_db_world/rev_1553294024713457200.sql

```diff
@@ -6,4 +6,4 @@ DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 7750);
 INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
 (7750, 0, 0, 0, 20, 0, 100, 0, 2701, 0, 0, 0, 41, 3000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, ''),
 (7750, 0, 1, 2, 19, 0, 100, 0, 2701, 0, 0, 0, 70, 0, 0, 0, 0, 0, 0, 14, 44733, 141980, 0, 0, 0, 0, 0, ''),
-(7750, 0, 2, 0, 61, 0, 100, 0, 0, 0, 0, 0, 70, 0, 0, 0, 0, 0, 0, 14, 44732, 141981, 0, 0, 0, 0, 0, '');```
```

Comment:
@Knindzagxg those are only supposed to be used on github comments, not in real code :)